lib/webauto/web4cucumber.rb

```diff
@@ -556,10 +556,11 @@ def handle_element(element_rule, **user_opts)
         element = elements.first.first.last
         opres = handle_operation(element, op, **user_opts)
 
-        if opres[:response].include? "#<Watir::Exception::UnknownObjectException: unable to locate element"
+        if opres[:response].include? "Watir::Exception::LocatorException"
```

Comment:
you think we never hit `UnknownObjectException` ?
Maybe we get it with Firefox?

It would be safer to keep check for it. But it is also fine if we remove it as you did above and we can revert back in case we hit that error in the future.

Reply:
@akostadinov  good point, it would be safer to keep it, let me revert the changes